src/components/CaVsNy.js

```diff
@@ -5,70 +5,64 @@ import axios from "axios";
 // Visualization for both CA and NY
 const CaVsNy = () => {
     const [chartData, setChartData] = useState({});
-    
+
     const chart = () => {
-      let filteredCA, filteredNy= [];  
+      let filteredCA, filteredNy= [];
       let covidDates = [];
       let covidCaDeath = [];
       let covidNyDeath = [];
 
       axios
         .get("https://covidtracking.com/api/v1/states/daily.json")
         .then(res => {
-            const state = res.data 
+            const state = res.data
             filteredCA = state.filter(a => a.state===("CA"))
-            
-
-            for (const dataObj of filteredCA){
```

Comment:
Just consistent spacing here (at most one line within an object or class). And adding spaces before curly braces